libs/shared/src/dto/workflows/update-workflow.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { NotificationStepDto } from './workflow.dto';
+
+export interface IUpdateWorkflowDto {
+  name?: string;
+
+  tags?: string[];
+
+  description?: string;
+
+  identifier?: string;
+
+  critical?: boolean;
+
+  steps?: NotificationStepDto[];
+
+  notificationGroupId?: string;
```

Comment:
Leaving the notificationGroupId as it is, will address this in a different PR where we will create new `workflowGroup` controllers.